src/istgt_lu_ctl.c

```diff
@@ -42,6 +42,7 @@
 #endif
 #include <unistd.h>
 #include <sys/param.h>
+#include <json-c/json_object.h>
```

Comment:
Add `REPLICATION` macro protection since we are using JSON related API for replication feature only.